### PR TITLE
Move `torchlie` out of labs

### PR DESCRIPTION
**Pull request by luisenp**

### CODE REVIEW

File: tests/lie_tests/functional/test_so3.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import pytest
 import torch
 
-from tests.decorators import run_if_labs
+from tests.theseus_tests.decorators import run_if_labs
```
Comment (fantaosha) [inline tests/lie_tests/functional/test_so3.py:10]: Do we still need `run_if_labs` if `torchlie` has been moved out of labs?

File: examples/lie_labs.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import torch
```
Comment (fantaosha) [inline examples/lie_labs.py:1]: Do need rename this file? Maybe `trochlie` instead of `lie_labs`?

**Review by fantaosha — APPROVED** [review]
LGTM

File: tests/lie_tests/functional/test_so3.py (hunk 1 above)
Comment (luisenp) [reply tests/lie_tests/functional/test_so3.py:10]: We don't need it, but I had missed this at first. I removed them in later PR, [here](https://github.com/facebookresearch/theseus/pull/527/commits).

File: examples/lie_labs.py (hunk 2 above)
Comment (luisenp) [reply examples/lie_labs.py:1]: Yes, but after we merge all of this stuff, I have to fix the library import to `torchlie` in multiple places, so we can do that then.

**Pull request merged**